tests/lib/schema_validation_test.py

```diff
@@ -64,26 +64,28 @@
             }
         }
     ],
-    ROUTES: [
-        {
-            'table-name': 'main',
-            'table-id': 254,
-            'origin': 'static',
-            'metric': 100,
-            'destination': '0.0.0.0/0',
-            'next-hop-interface': 'eth0',
-            'next-hop-address': '192.0.2.1'
-        },
-        {
-            'table-name': 'main',
-            'table-id': 254,
-            'origin': 'static',
-            'metric': 100,
-            'destination': '::/0',
-            'next-hop-interface': 'eth0',
-            'next-hop-address': 'fe80::1'
-        },
-    ]
+    ROUTES: {
+        'config': [
+            {
+                'table-name': 'main',
+                'table-id': 254,
+                'metric': 100,
+                'destination': '0.0.0.0/0',
+                'next-hop-interface': 'eth0',
+                'next-hop-address': '192.0.2.1'
+            }
+        ],
+        'running': [
+            {
+                'table-name': '',
+                'table-id': -1,
```

Comment:
It cannot be negative as any route belongs to some table.

Reply:
Fixed.

Reply:
Still negative.